Remove stale reference to the removed Generator.Name_Set package.

Commit d0b8dbdfe3 ("Remove package Generator.Name_Set", 2025-01-27)
deleted tools/wsdl2aws-generator-name_set.adb but left a reference to it
in tools.gpr's Locally_Removed_Files list. gprbuild now fails while
processing tools.gpr:

    tools.gpr:57:53: unknown file "wsdl2aws-generator-name_set.adb"

Drop the dangling entry.

--- a/tools/tools.gpr
+++ b/tools/tools.gpr
@@ -42,7 +42,6 @@
                for Locally_Removed_Files use Project'Locally_Removed_Files &
                  ("wsdl2aws.ads", "wsdl2aws-generator.adb",
                   "wsdl2aws-generator.ads",  "wsdl2aws-generator-cb.adb",
-                  "wsdl2aws-generator-name_set.adb",
                   "wsdl2aws-generator-skel.adb", "wsdl2aws-generator-stub.adb",
                   "wsdl2aws-main.adb", "wsdl2aws-wsdl-parameters.adb",
                   "wsdl2aws-wsdl-parameters.ads", "wsdl2aws-wsdl-parser.adb",
